iogt/settings/base.py

```diff
@@ -428,3 +430,53 @@
     'wagtailcore.locale': ['language_code'],
     'iogt_users.user': ['username'],
 }
+
+SESSION_ENGINE = 'django.contrib.sessions.backends.cached_db'
+
+CACHE_BACKEND = os.environ.get('CACHE_BACKEND')
+if CACHE_BACKEND == 'db':
```

Comment:
Redis is our preferred cache backend, but my concern is that we may not be able to set up a Redis cluster in the production environment before this PR is merged and released, so a temporary solution _may_ be necessary.

Using `DatabaseCache` as a temporary solution is not very appealing to me because a new cache table needs to be created for every deployment.

Using `LocMemCache` creates an in-memory cache for each process, which could be quite memory-intensive and lead to unpredictable results due to caches not being synchronised.

The `FileBasedCache` may be best as a temporary solution because it only requires a readable and writable directory on the local filesystem, which would be automatically created if it does not exist. I'm assuming that it is shared by all processes and that it's performance would be worse than in-memory, but better than database caching. Would you agree?

Reply:
in `FileBasesCache` each cache value will be stored as a separate value and the user running the server has to have read and write access for it to work, see [here](https://docs.djangoproject.com/en/4.0/topics/cache/#filesystem-caching)

Reply:
After discussion with @ChrisMarsh82, it has been decided to have two options for cache backend:

- Redis
- No cache (bypass)

If necessary, an intermediate option can be implemented later. We should be prepared to release the changes in this PR without Redis infrastructure in place.

Reply:
if we don't specify any cache django by default uses Local-Memory Cache, see [here](https://docs.djangoproject.com/en/4.0/topics/cache/#local-memory-caching-1)

Reply:
I would like to have the option to not cache at all, just in case there is any trouble.

Reply:
let's discuss in a quick call